### DISCUSSION TITLE
欢迎访问 MoviePilot 交流社区!

### DISCUSSION THREAD

**Discussion by jxxghp** [jxxghp/MoviePilot#366] [公告]
👋 欢迎！我们希望您能够：

- 提出您感兴趣的问题。
- 分享想法。
- 与其他社区成员互动。
- 欢迎其他人并持开放态度。请记住，这是我们一起建设的社区 💪。

只有一个要求：文明发言，禁止任何形式的不友好，如发现将会封禁对该项目及对应讨论群组的访问！

**Comment by DDSRem** [comment]
自 2024年12月08日 起，MoviePilot v1.x 版本仅修复 bug，不再功能更新，故v1版本所有`Feature Request`将移至`Discussions`！
